.changeset/eight-forks-breathe.md

```diff
@@ -0,0 +1,8 @@
+---
+"@salt-ds/core": minor
```

Comment:
We might want to split this changeset so that the actual breaking change is only listed in labs